### PR TITLE
Add menu options to change the zoom level on the webkit component

### PR DESCRIPTION
**Pull request by andrecasteliano**
Creates two new menu options (Zoom In and Zoom Out) to control the zoom level on the webkitview component.

### CODE REVIEW

File: src/WebView.cpp
Comment (eneshecan) [inline src/WebView.cpp:136]: Please use `auto zoomLevel = ...`

File: src/WebView.cpp
Comment (eneshecan) [inline src/WebView.cpp:145]: Please use `auto zoomLevel = ...`

File: src/WebView.cpp
Comment (eneshecan) [inline src/WebView.cpp:146]: Please use brackets on the next line for convention consistency.

**Review by eneshecan — CHANGES_REQUESTED** [review]

**Comment by eneshecan** [comment]
Thanks a lot for this! This definitely looks worse but we get to keep the accelerators and imho fanciness about zoom is not a big deal.

**Comment by eneshecan** [comment]
Just remembered something; it's better we keep this in `Settings` but I can do that after merging.

File: resource/ui/MainWindow.ui
Comment (andrecasteliano) [reply resource/ui/MainWindow.ui:49]: I created a submenu for the zoom options

File: src/WebView.cpp
Comment (andrecasteliano) [reply src/WebView.cpp]: Done

File: src/WebView.cpp
Comment (andrecasteliano) [reply src/WebView.cpp]: Done

File: src/WebView.cpp
Comment (andrecasteliano) [reply src/WebView.cpp]: Done

File: src/WebView.cpp
Comment (andrecasteliano) [reply src/WebView.cpp]: Done

**Comment by andrecasteliano** [comment]
New settings key 'zoom_level' was added. Take a look later and let me know if it's ok.

**Comment by eneshecan** [comment]
Looks great! I will merge when I have some time.

**Pull request merged**
